index.bs

```diff
@@ -12,23 +12,55 @@ Abstract: This API defines an API surface for controlling how audio is rendered
 Markup Shorthands: css no, markdown yes
 </pre>
 
+<pre class=link-defaults>
+spec:html; type:dfn; for:/; text:browsing context
+</pre>
+
 # Introduction # {#introduction}
 
-People consume a lot of media (audio/video) and the Web is one of the primary means of consuming this type of content.
-However, media on the web does not integrate well with the platform.
-The Audio Session API helps to close the gap with platforms that have audio session/audio focus such as Android and iOS.
-This API will help by improving the audio-mixing of websites with native apps, so they can play on top of each other, or play exclusively.
+People increasingly consume media (audio/video) through the Web,
+which has become a primary channel for accessing this type of content.
+However, media on the Web often lacks seamless integration with underlying platforms.
+The Audio Session API addresses this gap by enhancing media handling
+across platforms that support audio session management or similar audio focus features.
+This API improves how web-based audio interacts with other apps,
+allowing for better audio mixing or exclusive playback, depending on the context,
+to provide a more consistent and integrated media experience across devices.
 
-Additionally, on some platforms the user agent will automatically manage the audio session for the site
-based on whether media elements are playing or not and which APIs are used for playing audio.
-In some cases this may not match user expectations, this API provides overrides to authors.
+Additionally, some platforms automatically manage a site's audio session
+based on media playback and the APIs used to play audio.
+However, this behavior might not always align with user expectations.
+This API allows developers to override the default behavior and gain more control over an audio session.
 
-# The {{AudioSession}} interface # {#audiosession-interface}
+# Concepts # {#concepts}
+
+An <dfn>audio session</dfn> represents the playback of audible media.
+An [=audio session=] can be of a particular [=audio session/type=], and be in a particular [=audio session/state=].
+An [=audio session=] manages the audio for a set of sources and sinks, named audio session <dfn data-lt="element" for="audio session">elements</dfn>.
+
+An audio session [=audio session/element=] has a number of properties:
+
+* A <dfn data-lt="default type" for="audio session">default type</dfn>, which is used to compute the [=audio session=] [=audio session/type=], in case of "{{AudioSessionType/auto}}".
+* An <dfn>audible flag</dfn>, which is either `true` if the element is playing/recording audio or `false` otherwise.
+
+An audio session [=audio session/element=] is an <dfn>audible element</dfn> if its [=audible flag=] is `true`.
```

Comment:
I believe we need to clarify how to set the audible flag to true. Does this flag accurately reflect the actual audio output? We haven't yet discussed how to bind a media element (or other audible sources) to an audio session, which makes it somewhat confusing to define what constitutes an audible session.

Perhaps we should start by defining the criteria for determining if a session is audible and outline which types of elements can be used as sources.

Reply:
Regarding audio sources, I believe it would be clearer to slightly mention them here and link to the section below. This approach will help readers understand the audible behavior associated with the actual source, rather than just the AudioSession element.

For the audible flag, I recommend creating a separate section to explain it comprehensively. Even in Section 6, "Audio Source and Sink Integration," the definition of "audible" is not adequately addressed.

For example, if a non-muted media element is playing a silent audio track or has its audio track volume set to near inaudible levels, should this element be considered audible?

Additionally, how should we define "audible" for non-media elements, such as AudioContext and Web Speech?

If you agree with this approach, we could file another issue and create a separate PR for it.

Reply:
I filed https://github.com/w3c/audio-session/issues/27 and https://github.com/w3c/audio-session/issues/28 to keep track of this.